cfgov/jinja2/v1/activity-log/_activity-list.html

```diff
@@ -23,12 +23,12 @@
                     {% if is_blog(post) %}
                         {{ category_slug.render('blog',
                                 page_url,
-                                'meta-header_left',
+                                'm-meta-header_left',
```

Comment:
Hmmm, that seems really hacky, you'd expect all the other stuff to be applied as well if you're scanning the code. Mind leaving a note to address this later and either add the `m-meta-header` class or create a new class for the float rule?